.github/workflows/maven.yml

```diff
@@ -0,0 +1,15 @@
+name: Verify
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'main'
+  pull_request:
+
+jobs:
+  build:
+    name: Verify
+    uses: jetty/.github/.github/workflows/maven-ci.yml@main
```

Comment:
Same as other reviews.
This should be using the versioned notation `@5` (to a release or tag).
The branch notations `@main` will break the maven caching.

Reply:
read here https://github.com/jetty/jetty-build-support/pull/1#discussion_r1384247263